scripts-available/CDB_GhostTables.sql

```diff
@@ -0,0 +1,117 @@
+-- Table to store the transaction id from DDL events to avoid multiple executions
+CREATE TABLE IF NOT EXISTS cartodb.cdb_ddl_execution(txid integer PRIMARY KEY, tag text);
+
+-- Enqueues a job to run Ghost tables linking process for the provided user_id
+CREATE OR REPLACE FUNCTION _CDB_LinkGhostTables(username text, db_name text, ddl_tag text) 
+RETURNS void
+AS $$
+  if not username:
+    return
+
+  if 'json' not in GD:
+    import json
+    GD['json'] = json
+  else:
+    json = GD['json']    
+
+  error = ''
+  tis_config = plpy.execute("select cartodb.CDB_Conf_GetConf('invalidation_service');")[0]['cdb_conf_getconf']
+  tis_config_dict = json.loads(tis_config) if tis_config else {}
+  tis_host = tis_config_dict.get('host', '127.0.0.1')
+  tis_port = tis_config_dict.get('port', 3142)
+  tis_timeout = tis_config_dict.get('timeout', 5)
+  tis_retry = tis_config_dict.get('retry', 5)
+      
+  client = GD.get('invalidation', None)
+
+  while True:
```

Comment:
Ph34r!

Reply:
I don't like it either, but it comes from a similar function and it works. Do you want me to use `this_retry` here instead?

Reply:
I'm not sure, that's why I put ph34r. We can let that as it is